### PR TITLE
added comment count

### PR DESCRIPTION
**Pull request by Rionagrace**

### CODE REVIEW

**Review by nc-rose-mullan — COMMENTED** [review]
Looking good!

File: api/models/topics.models.js [hunk 1]
```diff
@@ -23,9 +23,37 @@ function selectArticleById(article_id) {
...
+    articles.article_img_url, 
+    COUNT(comments.article_id) AS comment_count
+FROM 
+    articles
+INNER JOIN 
+    comments 
+ON 
+    articles.article_id = comments.article_id
+GROUP BY 
+    articles.author,
```
Comment (nc-nick-jones) [inline api/models/topics.models.js:45]: this group by clause is unnecessarily long! have another look at how to do this later

**Review by nc-nick-jones — COMMENTED** [review]
just 1 comment!  continue for now, but come back and have a look at this whenever you can.

File: api/models/topics.models.js (hunk 1 above)
Comment (Rionagrace) [reply api/models/topics.models.js:45]: I thought it was !! haha, have updated now, I had an earlier issue and for some reason that was the only way to resolve, have updated and it all works now

**Pull request merged**